### ISSUE TITLE
Allow to specify command line argument of actionlint

### ISSUE THREAD

**Issue by rhysd** [github/super-linter#1852]
**Is your feature request related to a problem? Please describe.**

[actionlint](https://github.com/rhysd/actionlint) provides `-ignore` option to ignore some errors. This is necessary to ignore some false positives or errors which a user is not interested in. However, super-linter currently does not provide a way to pass command line options to actionlint.

Related issue: https://github.com/rhysd/actionlint/issues/26

**Describe the solution you'd like**

I'd like to have the way to ignore some errors from actionlint. It'd be nice to have some environment variable for it.

**Describe alternatives you've considered**

N/A

**Additional context**

N/A

**Comment by rhysd** [comment]
This issue is stuck because of #1667

**Comment by ferrarimarco** [comment]
Reopening this one, given that #1667 was abandoned.

**Comment by vorotech** [comment]
Please reopen the issue. Atm there is no way to ignore an error which occur when using `workflow_call`
The `secrets.GITHUB_TOKEN` causes a false positive
```
property "github_token" is not defined in object type
```
See https://github.com/rhysd/actionlint/issues/87

**Comment by ferrarimarco** [comment]
@vorotech at least this one should be fixed in the next super-linter release that ships with the latest actionlint release.

**Comment by SoumayaMauthoorMOJ** [comment]
Any update on this issue? This would allow me to ignore the 'SC2016:' error which is linked with https://github.com/github/super-linter/issues/3100

**Comment by artempas** [comment]
can't wait to see it

**Comment by ferrarimarco** [comment]
There's some work already done in #1853. I'll have a look into that to recover what @rhysd did.

**Comment by rhysd** [comment]
@ferrarimarco I recreated a PR onto the latest main branch at #3451.

**Comment by rhysd** [comment]
Closing without fixed since the maintainer is not interested in this issue.

**Comment by ifyum** [comment]
ya  arreglaron  esto ?  Run psql -h localhost -p 4[9]
psql: warning: extra command-line argument "pass" ignored
Password:
